RetailCoder.VBE/Inspections/QuickFixes/PassParameterByReferenceQuickFix.cs

```diff
@@ -19,22 +19,19 @@ public PassParameterByReferenceQuickFix(ParserRuleContext context, QualifiedSele
 
         public override void Fix()
         {
-            var parameter = Context.GetText();
+            string pattern = "^\\s*" + Tokens.ByVal + "(\\s+)";
+            string rgxReplacement = Tokens.ByRef + "$1";
+            Regex rgx = new Regex(pattern);
 
-            var parts = parameter.Split(new char[]{' '},2);
-            if (1 != parts.GetUpperBound(0))
-            {
-                return;
-            }
-            parts[0] = parts[0].Replace(Tokens.ByVal, Tokens.ByRef);
-            var newContent = parts[0] + " " + parts[1];
+            var parameter = Context.GetText();
```

Comment:
Don't worry about the `TokenStreamRewriter`. You can get the position of the token from the `Context` object.  There's a good example in the `ReplaceObsoleteCommentMarkerQuickFix` - take a look at [this section](https://github.com/rubberduck-vba/Rubberduck/blob/next/RetailCoder.VBE/Inspections/QuickFixes/ReplaceObsoleteCommentMarkerQuickFix.cs#L25-L37). You can even ignore the part about getting the line count - the identifiers you're replacing will always be on a single line.  I usually find it helpful to set a breakpoint, debug a test that uses that code path, and then examine the tree in the locals window.